proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -189,3 +193,33 @@ message RemoveOrganizationMemberRequest {
 }
 
 message RemoveOrganizationMemberResponse {}
+
+message GetOrganizationSettingsRequest{
+  // The ID of the organization for which to get the settings.
+  string organization_id = 1;
+}
+
+message GetOrganizationSettingsResponse {
+  RepositoryRole repository_role = 1;
+  PluginRole plugin_role = 2;
+  TemplateRole template_role = 3;
+}
+
+message UpdateOrganizationSettingsRequest {
+  message RepositoryRoleMessage {
```

Comment:
I am struggling with the naming here, also not sure is it better to put these nested, or separately outside the request message

Reply:
We generally don't nest messages, so I would move this outside this definition. How about `OptionalRepositoryRole`?

Reply:
I would follow the wrapper type structure from the well known types (https://buf.build/beta/wellknowntypes/file/main/google/protobuf/wrappers.proto), which has `<xyz>Value` messages wrap something with a single `value` field. I think they can be defined besides the enums in the current `role.proto`.

Reply:
> I would follow the wrapper type structure from the well known types (https://buf.build/beta/wellknowntypes/file/main/google/protobuf/wrappers.proto), which has `<xyz>Value` messages wrap something with a single `value` field. I think they can be defined besides the enums in the current `role.proto`.

this sounds good to me!